extensions/2.0/Khronos/KHR_implicit_shapes/README.md

```diff
@@ -0,0 +1,147 @@
+
+Shapes are defined within a dictionary property in the glTF scene description file, by adding an `extensions` property to the top-level glTF 2.0 object and defining a `KHR_implicit_shapes` property with a `shapes` array inside it.
+
+Each shape defines a mandatory `type` property which designates the type of shape, as well as an additional structure which provides parameterizations specific to that type. The following example defines a sphere.
+
+```javascript
+"extensions": {
+    "KHR_implicit_shapes" : {
+        "shapes": [
+            {
+                "sphere": { "radius": 0.1 },
+                "type": "sphere"
+            }
+        ]
+    }
+}
+```
+
+To describe the geometry which represents the object, shapes must define at most one of the following properties:
+
+|              | Type     | Description                                                                                                                                                                                      |
+| ------------ | -------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| **plane**    | `object` | A plane centered at the origin in local space, with normal along the Y axis in local space, optionally with finite extents and may be double or single-sided.                                    |
+| **sphere**   | `object` | A sphere centered at the origin in local space.                                                                                                                                                  |
+| **box**      | `object` | An axis-aligned box centered at the origin in local space.                                                                                                                                       |
+| **cylinder** | `object` | A cylinder centered at the origin and aligned along the Y axis in local space, with potentially different radii at each end. A cone is a special case of cylinder when one of the radii is zero. |
+| **capsule**  | `object` | A capsule (cylinder with hemispherical ends) centered at the origin and defined by two "capping" spheres with potentially different radii, aligned along the Y axis in local space.              |
+
+When the `type` of a shape refers to a shape type defined by this extension, the shape must not supply a value for a different shape type e.g. a shape whose `type` is "box" must not have a defined `sphere` property. Similarly, if `type` does not reference a shape type declared by `KHR_implicit_shapes`, an extension or other mechanism should supply an alternate shape definition.
+
+By default, planes have infinite extents along their tangent and bitangent vectors, specified as X and Z in local space, respectively. A plane may optionally provide `sizeX` and `sizeZ` properties to specify a finite size along these axes.
```

Comment:
This text is unclear, is the size "along these axes" referring to the radius or the diameter? Also, it should clarify +X and +Z for the tangent vectors.